dev-lang/janet/janet-0.5.0.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Dynamic Lisp dialect and bytecode vm"
+DESCRIPTION="Janet is a functional and imperative programming language \
+and bytecode interpreter. It is a modern lisp, but lists are replaced by \
+other data structures with better utility and performance (arrays, tables, \
+structs, tuples). The language also supports bridging to native code written \
+in C, meta-programming with macros, and bytecode assembly."
+HOMEPAGE="https://janet-lang.org/"
+COPYRIGHT="2019 Calvin Rose and contributors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/janet-lang/janet/archive/v0.5.0.tar.gz"
+CHECKSUM_SHA256="f5c90122715869657427cbba62809a1d53eaaced2268e7d55202301d65a75076"
+SOURCE_FILENAME="janet-$portVersion.tar.gz"
+PATCHES="janet-0.5.0.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${portVersion%%.*}"
+
+PROVIDES="
+	janet$secondaryArchSuffix = $portVersion
+	cmd:janet
+	lib:libjanet = $libVersionCompat
```

Comment:
If you install in in $binDir and $libDir, shouldn't these contain $secondaryArchSuffix for janet_x86 on gcc2h?

Reply:
Yes, I forgot to add the secondary architecture stuff, which is why I closed it; guess that I didn't do it in time.

Reply:
Why close it, you could push new changes here :)

Reply:
I know; I just didn't want anyone reviewing the mistakes while I worked it out and tested it. It should be fine now, I think.